docs/Developers/Git_Basics.rst

```diff
@@ -62,8 +72,9 @@ First let's talk about remotes. To communicate with the outside world, git uses
 what are called remotes. These are repositories other than the one on your
 local disk which you can push your changes into (so that other people can see
 them) or pull from (so that you can get others changes).
-Now you should add a remote to your local machine so that you can ``pull`` and
-``push`` your commits. This can be simply done by using the command:
+Now you should add a remote of your fork to your local machine so that you can
+``pull`` and ``push`` your commits. This can be simply done by using the
+command:
 
 ::
 
```

Comment:
If they add it as "myfork" they will not have a origin (default) remote, which will confuse git